src/game/Object/ObjectMgr.cpp

```diff
@@ -8041,6 +8052,19 @@ bool PlayerCondition::IsValid(uint16 entry, ConditionType condition, uint32 valu
                 return false;
             }
         }
+        case CONDITION_GAMEOBJECT_IN_RANGE:
+        {
+            if (!sCreatureStorage.LookupEntry<GameObjectInfo>(value1))
```

Comment:
Yes, it is :) This is part of sync work, the commit fixing this is in the update queue!